t/complex/aliceblog.xml

```diff
@@ -8,12 +8,12 @@
 <document xmlns="http://dlmf.nist.gov/LaTeXML" class="ltx_authors_1line" prefix="cc: http://creativecommons.org/ns# dcterms: http://purl.org/dc/terms/ foaf: http://xmlns.com/foaf/0.1/ schema: http://schema.org/ xsd: http://www.w3.org/2001/XMLSchema#">
   <resource src="LaTeXML.css" type="text/css"/>
   <resource src="ltx-article.css" type="text/css"/>
+  <rdf about="" property="foaf:neighbor" resourceidref="fredblog"/>
   <title>Alice’s Blog</title>
   <creator role="author">
     <personname>Alice</personname>
   </creator>
   <rdf aboutlabelref="LABEL:blog:photos" property="cc:license" resourcelabelref="LABEL:license"/>
-  <rdf about="" property="foaf:neighbor" resourceidref="fredblog"/>
```

Comment:
Interesting that this moves a single `rdf` element (rather than none or all).

And in that vein - do we want to have a small `{titlepage}` test, just to make sure the order remains as expected?

Reply:
There needs to be both a digestion & construction component, but I was able to simplify to a single `\lx@frontmatterhere`.

But thanks for drawing my attention to the need to explain, which also had me realize that the semi-redundant abstract placement from (#1638) was not only redundant, but got broken.  So the next commits combine things much more cleanly.  However, you may want to recheck the article mentioned in #1638 (I can't get at it).

Also, added a couple of tests, thanks for mentioning.